release/cyclone-server.yaml

```diff
@@ -64,8 +64,8 @@ _config:
         modes:
         - ReadWriteMany
       storage:
-        request: 200Gi
-        limit: 200Gi
+        request: 500Gi
```

Comment:
Maybe the quota is too large as we will not allow oversold of storage.
/cc @iawia002

Reply:
/hold
We are going to discuss this.

Reply:
update it to 10Gi

Reply:
LGTM